### ISSUE TITLE
Incorrect display of roles

### ISSUE THREAD

**Issue by Flapchik** [BadgerCode/tttdamagelogs#26]
If the user connected during the round, then he is displayed in the list of roles.
![image](https://user-images.githubusercontent.com/58729268/103438900-46dacc80-4c49-11eb-9fe7-caa42cd51a18.png)
![image](https://user-images.githubusercontent.com/58729268/103438905-4e01da80-4c49-11eb-9cbc-7bab88c47478.png)

**Comment by BadgerCode** [comment]
Cheers! I'll look into this

**Comment by Yogpod** [comment]
damagelogs/client/listview.lua line 204 It looks like the roles start at 0 index instead of 1

```
local role_colors = {
    [0] = Color(0, 200, 0),
    [1] = Color(200, 0, 0),
    [2] = Color(0, 0, 200),
    ["disconnected"] = Color(0, 0, 0)
}
```

changing the table to this worked for my server

**Comment by BadgerCode** [comment]
Yeah I can confirm the colours are off
![colours](https://i.imgur.com/oxPOaEB.png)

It seems they're all off by one or something.

**Comment by BadgerCode** [comment]
This will be fixed in the next release

**Comment by Flapchik** [comment]
![image](https://user-images.githubusercontent.com/58729268/106356990-79ddb200-6314-11eb-84ad-d030827668e8.png)

**Comment by Flapchik** [comment]
Hello. I meant a different problem. 
When a user connects during a round, he is assigned the role of "innocent", although he is an spectator.
![image](https://user-images.githubusercontent.com/58729268/107531663-200ba080-6bce-11eb-8ba3-31b19e388c15.png)

**Comment by BadgerCode** [comment]
Ah I see. I will reopen this issue.

**Comment by BadgerCode** [comment]
Should be fixed in #47 
Will be included in the next release
